### ISSUE TITLE
Pre-trained model on imageNet-21k and fine-tuned on Stanford Dog Dataset

### ISSUE THREAD

**Issue by dgrnd4** [huggingface/blog#369]
Hi there, I am trying to Fine-Tuning the ViT pre-trained on imageNet-21k on Stanford Dog Dataset.

I have downloaded the dataset and divided into Tr and Te.
I create a Dict, trying to get the same shape of the example that you gave in the paper but I am not able to train it.

Do you know if there is a way to download the dataset as you did with the " Beans " dataset?
do you know if there is a way to do something like load_dataset("Stanford_dog_dataset")

Many thanks,
it would be really helpful!

**Comment by mariosasko** [comment]
Hi! You can use `Dataset.from_dict(dict)` to convert the dictionary to an HF dataset object. This dataset is not on the Hub yet, so `load_dataset("Stanford_dog_dataset")` won't work, but feel free open an issue in the datasets repo to request its addition.

**Comment by dgrnd4** [comment]
@mariosasko thank you for the fast reply.

What I am wondering is that in the paper it has a key ['train'], used like ds['train'] that returns:
`
{
  'pixel_values': tensor([[[[-0.6157, -0.6000, -0.6078,  ..., ]]]]),`

`
  'labels': [0, 0]
}
`
I'm working with the Test Set because it's smaller than the Training Set.

When I do `arr_transf_te = transform(x_te,y_te)`, then I `print(arr_transf_te)` and get:
`
{
'pixel_values': tensor([[[[ 0.3882,  0.5373,  0.6627,  ...,  0.5294,  0.5137,  0.5216],
          [ 0.5373,  0.7333,  0.7725,  ...,  0.5451,  0.4431,  0.4275],
          [ 0.6314,  0.7333,  0.7255,  ...,  0.3882,  0.3804,  0.3647],
          ...,
``
'labels': [0,0,0,....,119,119]
}
`

after, `arr_transf_te = Dataset.from_dict(arr_transf_te)` and if I print it I obtain:
`
Dataset({
    features: ['pixel_values', 'labels'],
    num_rows: 4160
})
`

**Comment by dgrnd4** [comment]
@mariosasko what I should have is something like:

```
DatasetDict({
    train: Dataset({
           features: ['pixel_values'], ['labels'],
           num_rows:4160
})
```
That I should obtain from the one that I have now:
```
Dataset({ 
        features: ['pixel_values', 'labels'],
        num_rows: 4160
 })
```
 So I have to add the ['train'] key: any recommendation on how to do that?
